package.json

```diff
@@ -33,31 +33,31 @@
     "postinstall": "ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points"
   },
   "dependencies": {
-    "@angular/animations": "9.1.4",
-    "@angular/common": "9.1.4",
-    "@angular/compiler": "9.1.4",
-    "@angular/core": "9.1.4",
-    "@angular/forms": "9.1.4",
-    "@angular/platform-browser": "9.1.4",
-    "@angular/platform-browser-dynamic": "9.1.4",
-    "@angular/router": "9.1.4",
-    "@clr/angular": "3.1.5",
+    "@angular/animations": "10.0.14",
+    "@angular/common": "10.0.14",
+    "@angular/compiler": "10.0.14",
+    "@angular/core": "10.0.14",
+    "@angular/forms": "10.0.14",
+    "@angular/platform-browser": "10.0.14",
+    "@angular/platform-browser-dynamic": "10.0.14",
+    "@angular/router": "10.0.14",
+    "@clr/angular": "4.0.1",
     "@clr/core": "3.1.5",
```

Comment:
this also needs an update... did you update by hand or with "ng update..."? later one is recommended, so the migration scripts of angular and clarity get executed. so if you did by hand, pls run those scripts to be sure.

Reply:
Thx - forgot this again. 

I did use `ng update`

Reply:
also forgot to adapt the package.json in npm/clr-addons - will fix that

Reply:
Fixed in current commit